### PR TITLE
Add missing MODs for Arasy Weapons

### PR DESCRIPTION
**Pull request by neuromancerxi**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Adds MODs (and applicable Pet MODs) for:

Arasy Knife / +1
Arasy Sword / +1
Arasy Claymore / +1
Arasy Tabar / +1

## Steps to test these changes

Change to Relevant Job for each Item (WAR, THF, etc)
Add 50+ Job Points
Add Key Item `JOB_BREAKER`
Spend 50+ Job Points to unlock SU1
Equip Item
Check MODs

### CODE REVIEW

**Review by EpicTaru — APPROVED** [review]

File: sql/item_mods.sql [hunk 1]
```diff
@@ -43991,6 +44035,20 @@ INSERT INTO `item_mods` VALUES (21700,25,20); -- ACC: 20
...
+INSERT INTO `item_mods` VALUES (21704,25,10);   -- ACC: 10
+INSERT INTO `item_mods` VALUES (21704,345,100); -- TP_BONUS: 100
+
+-- Arasy Tabar +1
+INSERT INTO `item_mods` VALUES (21705,8,11);    -- STR: 11
+INSERT INTO `item_mods` VALUES (21705,9,11);    -- DEX: 11
+INSERT INTO `item_mods` VALUES (21705,11,11);   -- AGI: 11
+INSERT INTO `item_mods` VALUES (21705,25,15);   -- ACC: 15
+INSERT INTO `item_mods` VALUES (21705,345,150); -- TP_BONUS: 150
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:44051]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21705,84,242);   -- AXE: 242
INSERT INTO `item_mods` VALUES (21705,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21705,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 2]
```diff
@@ -43991,6 +44035,20 @@ INSERT INTO `item_mods` VALUES (21700,25,20); -- ACC: 20
 -- Blurred Claymore +1
 INSERT INTO `item_mods` VALUES (21701,25,25); -- ACC: 25
 
+-- Arasy Tabar
+INSERT INTO `item_mods` VALUES (21704,8,6);     -- STR: 6
+INSERT INTO `item_mods` VALUES (21704,9,6);     -- DEX: 6
+INSERT INTO `item_mods` VALUES (21704,11,6);    -- AGI: 6
+INSERT INTO `item_mods` VALUES (21704,25,10);   -- ACC: 10
+INSERT INTO `item_mods` VALUES (21704,345,100); -- TP_BONUS: 100
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:44044]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21704,84,242);   -- AXE: 242
INSERT INTO `item_mods` VALUES (21704,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21704,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 3]
```diff
@@ -43851,6 +43881,20 @@ INSERT INTO `item_mods` VALUES (21633,30,50);   -- MACC: 50
...
+INSERT INTO `item_mods` VALUES (21654,73,3);    -- STORETP: 3
+INSERT INTO `item_mods` VALUES (21654,841,2);   -- ALL_WSDMG_FIRST_HIT: 2
+
+-- Arasy Claymore +1
+INSERT INTO `item_mods` VALUES (21655,8,17);    -- STR: 17
+INSERT INTO `item_mods` VALUES (21655,9,17);    -- DEX: 17
+INSERT INTO `item_mods` VALUES (21655,23,20);   -- ATT: 20
+INSERT INTO `item_mods` VALUES (21655,73,5);    -- STORETP: 5
+INSERT INTO `item_mods` VALUES (21655,841,3);   -- ALL_WSDMG_FIRST_HIT: 3
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43897]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21655,83,242);   -- GSWORD: 242
INSERT INTO `item_mods` VALUES (21655,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21655,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 4]
```diff
@@ -43851,6 +43881,20 @@ INSERT INTO `item_mods` VALUES (21633,30,50);   -- MACC: 50
 INSERT INTO `item_mods` VALUES (21633,122,20);  -- BLUE: 20
 INSERT INTO `item_mods` VALUES (21633,311,217); -- MAGIC_DAMAGE: 217
 
+-- Arasy Claymore
+INSERT INTO `item_mods` VALUES (21654,8,12);    -- STR: 12
+INSERT INTO `item_mods` VALUES (21654,9,12);    -- DEX: 12
+INSERT INTO `item_mods` VALUES (21654,23,15);   -- ATT: 15
+INSERT INTO `item_mods` VALUES (21654,73,3);    -- STORETP: 3
+INSERT INTO `item_mods` VALUES (21654,841,2);   -- ALL_WSDMG_FIRST_HIT: 2
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43890]:
```sql
INSERT INTO `item_mods` VALUES (21654,83,242);   -- GSWORD: 242
INSERT INTO `item_mods` VALUES (21654,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21654,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 5]
```diff
@@ -43725,6 +43741,20 @@ INSERT INTO `item_mods` VALUES (21584,26,50);   -- RACC: 50
...
+INSERT INTO `item_mods` VALUES (21604,25,10);   -- ACC: 10
+INSERT INTO `item_mods` VALUES (21604,291,3);   -- COUNTER: 3
+
+-- Arasy Sword +1
+INSERT INTO `item_mods` VALUES (21605,9,11);    -- DEX: 11
+INSERT INTO `item_mods` VALUES (21605,10,11);   -- VIT: 11
+INSERT INTO `item_mods` VALUES (21605,13,11);   -- MND: 11
+INSERT INTO `item_mods` VALUES (21605,25,15);   -- ACC: 15
+INSERT INTO `item_mods` VALUES (21605,291,5);   -- COUNTER: 5
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43757]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21605,82,242);   -- SWORD: 242
INSERT INTO `item_mods` VALUES (21605,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21605,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 6]
```diff
@@ -43725,6 +43741,20 @@ INSERT INTO `item_mods` VALUES (21584,26,50);   -- RACC: 50
 INSERT INTO `item_mods` VALUES (21584,30,50);   -- MACC: 50
 INSERT INTO `item_mods` VALUES (21584,311,217); -- MAGIC_DAMAGE: 217
 
+-- Arasy Sword
+INSERT INTO `item_mods` VALUES (21604,9,6);     -- DEX: 6
+INSERT INTO `item_mods` VALUES (21604,10,6);    -- VIT: 6
+INSERT INTO `item_mods` VALUES (21604,13,6);    -- MND: 6
+INSERT INTO `item_mods` VALUES (21604,25,10);   -- ACC: 10
+INSERT INTO `item_mods` VALUES (21604,291,3);   -- COUNTER: 3
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43750]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21604,82,242);   -- SWORD: 242
INSERT INTO `item_mods` VALUES (21604,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21604,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 7]
```diff
@@ -43595,6 +43595,22 @@ INSERT INTO `item_mods` VALUES (21528,25,40); -- ACC: 40
...
+INSERT INTO `item_mods` VALUES (21554,165,2);   -- CRITHITRATE: 2
+
+-- Arasy Knife +1
+INSERT INTO `item_mods` VALUES (21555,9,11);    -- DEX: 11
+INSERT INTO `item_mods` VALUES (21555,11,11);   -- AGI: 11
+INSERT INTO `item_mods` VALUES (21555,14,11);   -- CHR: 11
+INSERT INTO `item_mods` VALUES (21555,25,15);   -- ACC: 15
+INSERT INTO `item_mods` VALUES (21555,31,27);   -- EVA: 27
+INSERT INTO `item_mods` VALUES (21555,165,3);   -- CRITHITRATE: 3
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43613]: Missing 
```sql
INSERT INTO `item_mods` VALUES (21555,81,242);   -- DAGGER: 242
INSERT INTO `item_mods` VALUES (21555,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21555,30,188);   -- MACC: 188
```

File: sql/item_mods.sql [hunk 8]
```diff
@@ -43595,6 +43595,22 @@ INSERT INTO `item_mods` VALUES (21528,25,40); -- ACC: 40
...
 INSERT INTO `item_mods` VALUES (21529,25,40); -- ACC: 40
 
+-- Arasy Knife
+INSERT INTO `item_mods` VALUES (21554,9,6);     -- DEX: 6
+INSERT INTO `item_mods` VALUES (21554,11,6);    -- AGI: 6
+INSERT INTO `item_mods` VALUES (21554,14,6);    -- CHR: 6
+INSERT INTO `item_mods` VALUES (21554,25,10);   -- ACC: 10
+INSERT INTO `item_mods` VALUES (21554,31,22);   -- EVA: 22
+INSERT INTO `item_mods` VALUES (21554,165,2);   -- CRITHITRATE: 2
+
```
Comment (Mortalelite) [inline sql/item_mods.sql:43605]: Missing
```sql
INSERT INTO `item_mods` VALUES (21554,81,242);   -- DAGGER: 242
INSERT INTO `item_mods` VALUES (21554,110,242);   -- PARRY: 242
INSERT INTO `item_mods` VALUES (21554,30,188);   -- MACC: 188
```

Comment (Xaver-DaRed) [reply sql/item_mods.sql:43605]: "Magic Accuracy Skill" is not the same as "Magic Accuracy"

Same for all the ones above. I'm that lazy.

File: sql/item_mods.sql (hunk 5 above)
Comment (TeoTwawki) [reply sql/item_mods.sql:43757]: @Mortalelite I've taken the liberty of wrapping your sql example in the proper tags so that it parses properly

**Comment by neuromancerxi** [comment]
@Mortalelite: ilvl Weapon Skill, Parry, and MACC are handled in `item_weapon` not `item_mods`

https://github.com/LandSandBoat/server/blob/f750a71effffc4ba9728387a90c7beacc2a08856/sql/item_weapon.sql#L24

e.g. 
https://github.com/LandSandBoat/server/blob/f750a71effffc4ba9728387a90c7beacc2a08856/sql/item_weapon.sql#L4432

File: sql/item_mods.sql (hunk 5 above)
Comment (Mortalelite) [reply sql/item_mods.sql:43757]: oh it all good

**Comment by Mortalelite** [comment]
> @Mortalelite: ilvl Weapon Skill, Parry, and MACC are handled in `item_weapon` not `item_mods`
> 
> https://github.com/LandSandBoat/server/blob/f750a71effffc4ba9728387a90c7beacc2a08856/sql/item_weapon.sql#L24
> 
> e.g.
> 
> https://github.com/LandSandBoat/server/blob/f750a71effffc4ba9728387a90c7beacc2a08856/sql/item_weapon.sql#L4432

Ok if that the case then disregard my comments

File: sql/item_mods_pet.sql [hunk 9]
```diff
@@ -537,6 +537,11 @@ INSERT INTO `item_mods_pet` VALUES (21529,25,40,0); -- All Pets - ACC: 40
 INSERT INTO `item_mods_pet` VALUES (21529,26,40,0); -- All Pets - RACC: 40
 INSERT INTO `item_mods_pet` VALUES (21529,30,40,0); -- All Pets - MACC: 40
 
+-- Arasy Tabar
+INSERT INTO `item_mods_pet` VALUES (21704,25,10,0); -- All Pets - ACC: 10
+
+-- Arasy Tabar +1
+INSERT INTO `item_mods_pet` VALUES (21705,25,15,0); -- All Pets - ACC: 15
```
Comment (TeoTwawki) [inline sql/item_mods_pet.sql:544]: need a line break here before the `-- Monster Axe` line

Comment (neuromancerxi) [reply sql/item_mods_pet.sql:544]: Nice catch, done.

**Review by Xaver-DaRed — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**